### DISCUSSION TITLE
Vague quiz in Lesson 2: Healthy Market Competition for Information Services

### DISCUSSION THREAD

**Discussion by jaintj95** [OpenMined/courses#87] [Feedback] [answered]
Quiz 3: Privacy is about preventing information from being shared in inappropriate contexts.

Selecting **True** is considered as an incorrect answer but please note that the question did not ask if "Privacy is **only** about preventing ..... contexts" 
In the current case, the question's interpretation is slightly vague. I think clarifying it further would help reduce confusion :)

**Comment by em-blue** [comment, marked as answer]
Good catch, just updated it!

**Reply by iamtrask** [reply]
Thank you @jaintj95 and @em-blue
